components/twitter/actions/upload-media/upload-media.mjs

```diff
@@ -41,7 +41,7 @@ export default {
     });
 
     // APPEND
-    const size = 5012;
+    const size = 8000;
```

Comment:
As per [the docs](https://developer.twitter.com/en/docs/tutorials/uploading-media) the max chunk size is 1MB, should we use this value?

```suggestion
    const size = 1000000;
```

Reply:
@andrewjschuang I found that setting the value higher resulted in an error saying that the request headers were too large.

Reply:
Higher than 8000?